peft/fft.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
```

Comment:
I feel like different .sh scripts are not needed, one script with command line options is good enough. or we can not check in these scripts and have it as example usage in the header comment of finetune.py

Reply:
Okay, I will keep only one of them. 

Running `python3 finetune.py --help` gives the following o/p, which is self-explanatory. Perhaps I can add it in README?
![image](https://github.com/user-attachments/assets/d388a6d7-3066-437c-ae04-a83fc32e73ca)